misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.utils import timezone
+from django.utils.six import StringIO
+from django.test import TestCase, override_settings
+
+from misago.core.utils import ANONYMOUS_IP
+from misago.users.management.commands import anonymizeoldips
+
+
+UserModel = get_user_model()
+
+
+class AnonymizeOldIpsTests(TestCase):
+    def test_anonymizeoldips_new_user(self):
+        """command is not anonymizing user's IP if its new"""
+        user = UserModel.objects.create_user('Bob', 'bob@bob.com')
+        call_command(anonymizeoldips.Command())
+        user_joined_from_ip = UserModel.objects.get(pk=user.pk).joined_from_ip
+        
+        self.assertNotEqual(user_joined_from_ip, ANONYMOUS_IP)
```

Comment:
Lets remove this blank line so value definition and test are right next to each other. Instead please add blank line around the `call_command` line.

Reply:
u mean to say like this?
```
def test_removeoldips_old_user(self):
        """command is anonymizing user's IP if its old"""
        joined_on_past = timezone.now() - timedelta(days=50)
        user = UserModel.objects.create_user('Bob1', 'bob1@bob.com')
        user.joined_on = joined_on_past
        user.save()

        call_command(removeoldips.Command())
        user_joined_from_ip = UserModel.objects.get(pk=user.pk).joined_from_ip
        self.assertEqual(user_joined_from_ip, ANONYMOUS_IP)
```